File: hadasim/src/app/classes/Patient.ts
export class Patient{
    constructor (
        public Id?:string,
        public FirstName?:string,
        public LastName?:string,
        public AddressId?:number,
        public BurnDate?:Date,
        public Phone?:string,
        public Tel?:string,
        public Pic?:string,
        public Status?:boolean
    ){}
}

File: hadasim/HMO_corona/DAL/HMO/Patient.cs
using System;
using System.Collections.Generic;

namespace DAL.HMO;

public partial class Patient
{
    public string Id { get; set; } = null!;

    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public int? AddressId { get; set; }

    public DateTime? BurnDate { get; set; }

    public string? Phone { get; set; }

    public string? Tel { get; set; }

    public string? Pic { get; set; }

    public bool? Status { get; set; }

    public virtual PatientAddress? Address { get; set; }
}
